### DISCUSSION TITLE
Image attachments to draft issues is a must

### DISCUSSION THREAD

**Discussion by lschnoller** [community/community#10545] [Projects and Issues]
In order to use the new Projects Beta, it's a must to be able to attach images to draft issues. Our less tech savy users love using Project Beta because it's not attached to a specific repo (like the regular project). This is great for non developers to track their non repo work and also to report issues with the app without having to know exactly to what repo the issue belongs to. The problem is that they can't attach screenshots so they can't report issues properly. This issue made us have to go back to trello.

**Comment by RomeHein** [comment]
💯
In addition I would say we should be able to add any type of attachment: image/gif/small video/figma/excel/google doc and so on.

**Comment by lukaszflorczak** [comment]
It should be similar like on Trello. There, that is super easy.

**Reply by rmcsharry** [reply]
> It should be similar like on Trello. There, that is super easy.

![image](https://user-images.githubusercontent.com/1156100/162616250-b2d46633-5742-4ffd-8039-16333d325873.png)

**Comment by dcrouch79** [comment]
Found a workaround:

1. Open any repo (doesn't matter which)
2. Click "New Issue" and paste in the screenshot. Copy the auto-generated url of the form `https://user-images.githubusercontent.com/.../...png`. The new issue can now be cancelled.
3. Add image using standard image markdown: `![image](https://user-images.githubusercontent.com/.../...png)`

FYI, the only reason this works is that images uploaded via repo issues are not private. Even if the repo is private! https://wishmesh.com/2017/03/attachments-from-githubs-private-issue-trackers-can-be-viewed-without-any-authentication/

**Comment by rmcsharry** [comment]
> In order to use the new Projects Beta, it's a must to be able to attach images to draft issues. Our less tech savy users love using Project Beta because it's not attached to a specific repo (like the regular project). This is great for non developers to track their non repo work and also to report issues with the app without having to know exactly to what repo the issue belongs to. The problem is that they can't attach screenshots so they can't report issues properly. This issue made us have to go back to trello.

![image](https://user-images.githubusercontent.com/1156100/162616259-cacb4750-0ce4-4645-ad2b-1ea7afa465dd.png)
